### DISCUSSION TITLE
hc don't change to night.

### DISCUSSION THREAD

**Discussion by JokerGermany** [emsesp/EMS-ESP32#1096] [General]
![grafik](https://user-images.githubusercontent.com/30293477/222881857-0cd2e24e-68a0-4205-9ff8-ba30c6070e35.png)
![grafik](https://user-images.githubusercontent.com/30293477/222881877-b7ee18b4-1a1d-4a27-9077-8a36d2b9c631.png)

![grafik](https://user-images.githubusercontent.com/30293477/222881922-7c5de431-443e-4b51-9846-cd05b02e1c9f.png)
![grafik](https://user-images.githubusercontent.com/30293477/222881935-9cd8730b-5048-4f0b-825d-d6f21c3151f8.png)

Locks like the option isn't transfered to ems-esp?

**Comment by proddy** [comment]
which thermostat are you using?

**Reply by JokerGermany** [reply]
RC30

**Reply by proddy** [reply]
device_id? attaching the support info would help

**Reply by JokerGermany** [reply]
Wil do when the EMS-ESP Webinterface is available again.

**Reply by MichaelDvP** [reply]
This is a bug introduced with #786. In night mode byte 0 of 3D goes to zero, we have to check the roomtemp instead, this is valid or 7D00 for active hc's.

**Reply by JokerGermany** [reply]
[emsesp_info.txt](https://github.com/emsesp/EMS-ESP32/files/10889223/emsesp_info.txt)

**Reply by JokerGermany** [reply]
If you wonder because of the uptime, i installed the latest dev version before
